operations/iuf/workflows/configuration.md

```diff
@@ -176,26 +178,29 @@ Once this step has completed:
 ## 3. Perform manual product configuration operations
 
 Some products must be manually configured prior to the creation of CFS configurations and images. The "Install and Upgrade Framework" section of each individual product's installation documentation contains instructions for product-specific
-configuration, if any. The following highlights some of the areas that most often require manual configuration changes **but is not intended to be a comprehensive list.** Note that many of the configuration changes are only
+configuration, if any. Major changes may also be documented in the _HPE Cray EX System Software Stack Installation and Upgrade Guide for CSM (S-8052)_.
+The following highlights some of the areas that require manual configuration changes **but is not intended to be a comprehensive list.** Note that many of the configuration changes are only
 required for initial installation scenarios.
 
-- Products that most often require site customizations to Ansible content in VCS
-  - COS
-  - CPE
-  - Slingshot Host Software
-  - UAN
-  - Slurm and PBS workload managers
-- Initial install configuration changes (not required for upgrade scenarios)
-  - SAT
-    - Configure SAT authentication via `sat auth`
-    - Generate SAT S3 credentials
-    - Configure system revision information via `sat setrev`
-  - SDU
-    - Configure SDU via `sdu setup`
-  - COS
-    - Set the COS root password in HashiCorp Vault
-  - UAN
-    - Set the UAN root password in HashiCorp Vault
+- COS
+  - Configure DVS and LNet with appropriate Slingshot settings
+  - Configure DVS and LNet for use on application nodes
```

Comment:
Maybe we should have say something like this.
* Configure DVS and LNet for use on application nodes (this is a change for this release)